Unittest cleanup functions are called even if a test fails.
